codice/ddf-ui
# Refactored Email Notification configuration and scheduler

#### What does this PR do?

This PR fixes email notification scheduling and configuration update
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@lcrosenbu @glenhein 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@andrewkfiedler
@jlcsmith 
#### How should this be tested?

Build and Install DDF, configure admin to use a valid email, configure email notification and query monitor, create a workspace with standing queries and subscribe to it, and observe emails are being sent out at the proper interval.
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:
